src/app/visual/angular-tree-component/angular-tree-visual.component.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  ChangeDetectionStrategy,
+  Component
+} from '@angular/core';
+
+import {
+  TreeNode,
+  ITreeOptions
+} from 'angular-tree-component';
+
+import {
+  IDTypeDictionary
+} from 'angular-tree-component/dist/defs/api';
+
+@Component({
+  selector: 'sky-angular-tree-visual',
+  templateUrl: './angular-tree-visual.component.html',
+  styleUrls: ['./angular-tree-visual.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SkyAngularTreeVisualComponent {
+
+  public nodes: any[] = [
```

Comment:
I've noticed in a few places that we just type the stuff about `nodes` as `any`. Is there no object type we can use for any of these?

Reply:
It would be nice, but the library just defines this property as an `any` type as well. If you try to make it a `TreeNode` type, it complains about a lot of properties missing/wrong. `any` is the way the docs tell you to do it, so I just followed suit here.

Reply:
Ok :(